src/screens/AddSiteFormScreen.js

```diff
@@ -335,67 +337,75 @@ class AddSiteFormScreen extends Component {
                         {site_info}
                     </Text>
 
-                    <FormLabel>{title}</FormLabel>
-                    <FormInput
+                    <Text style={labelStyle}>{title}</Text>
+                    <Input
+                        autoCapitalize={'words'}
```

Comment:
Nice little helper that I've been meaning to add... the site's `Title` is now always auto-capitalized